crypto/fipsmodule/rsa/rsa.c

```diff
@@ -1045,9 +1045,18 @@ int RSA_flags(const RSA *rsa) { return rsa->flags; }
 int RSA_test_flags(const RSA *rsa, int flags) { return rsa->flags & flags; }
 
 int RSA_blinding_on(RSA *rsa, BN_CTX *ctx) {
+  if (rsa != NULL) {
+    rsa->flags &= (~RSA_FLAG_NO_BLINDING);
+  }
```

Comment:
this will not work, there is a setup routine that should be done for blinding to be enabled. I suggest leaving the function a no-op.

Reply:
If `rsa_blinding_off_temp_for_accp_compatibility` really does disable blinding and there's no way for bliding to be re-enabled, then `RSA_blinding_on` should return an error (`0`) when it's later called (instead of returning success on the no-op).